app/__init__.py

```diff
@@ -8,9 +8,11 @@
 
 # Initialize our application
 app = Flask(__name__, template_folder="templates")
+# debug = os.getenv('FLASK_DEBUG', False)
+env = os.environ.get('APP_ENV', 'development')
```

Comment:
This needs to default to production, or else it won't work in production where we don't use the environment variables.

Reply:
Great catch. Thanks.